cloudinary-image-management-and-manipulation-in-the-cloud-cdn/instance.php

```diff
@@ -7,25 +7,23 @@
 
 namespace Cloudinary;
 
-global $cloudinary_plugin;
-
-if ( ! defined( 'DEBUG_SCRIPTS' ) ) {
-	define( 'CLDN_ASSET_DEBUG', '.min' );
-} else {
-	define( 'CLDN_ASSET_DEBUG', '' );
-}
+define( 'CLDN_ASSET_DEBUG', defined( 'DEBUG_SCRIPTS' ) ? '' : '.min' );
 
+require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
You have to comment out this line to run static analysis.

Actually `get_plugin_data()` reads **hard-coded** data from `cloudinary.php` :)